marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/mapping/mapping-lib.sjs

```diff
@@ -0,0 +1,32 @@
+/**
+ Copyright (c) 2021 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+
+function parseMappingError(e){
+  let error;
```

Comment:
Assuming that this function should only return a "string message" as opposed to either a string or an error object, this should be renamed to "message".